pipeline-scripts/buildlib.groovy

```diff
@@ -728,6 +728,37 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
     }
 }
 
+def sync_images(major, minor, mail_list) {
+    // Run an image sync after a build. This will mirror content from
+    // internal registries to quay. After a successful sync an image
+    // stream is updated with the new tags and pullspecs.
+    if(major < 4)
+        return
+    final param = { type, name, value ->
+        [$class: type + 'ParameterValue', name: name, value: value]
+    }
+    final full_version = "${version}-${release}"
+    try {
+        build(job: 'build%252Fbuild-sync', parameters:
```

Comment:
Ooo... sneaky. I like it.

Reply:
lol. I copied it from the build ami's one.